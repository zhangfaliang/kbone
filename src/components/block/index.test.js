import toJson from 'enzyme-to-json';
import { render } from 'enzyme';
import Block from './index';

describe('Block', async () => {
  it('base', () => {
    expect.assertions(1);
    const wrapper = render(<Block />);
    expect(toJson).toMatchSnapshot();
  });
});
