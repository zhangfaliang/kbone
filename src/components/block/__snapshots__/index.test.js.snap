// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Block base 1`] = `[Function]`;
